src/fhirConfig.ts

```diff
@@ -160,4 +161,5 @@ export interface FhirConfig {
     server: Server;
     logging: Logging;
     profile: Profile;
+    validators: Validator[];
```

Comment:
looks good, just a couple things:

This is a breaking change. Do make sure to include `BREAKING CHANGE` on the commit message footer so that the version is bumped properly.

Also, we may want to hold on before merging this until we have the rest of the validator refactoring ready. This is to prevent us from being unable to release other interface changes in the meantime.

Reply:
Sounds good. I'll add that message when merging this PR in.